.dockstore.yml

```diff
@@ -158,6 +158,16 @@ workflows:
          - ah_var_store
        tags:
          - /.*/
+   - name: GvsValidateVDS
+     subclass: WDL
+     primaryDescriptorPath: /scripts/variantstore/wdl/GvsValidateVDS.wdl
+     filters:
+       branches:
+         - master
+         - ah_var_store
+         - EchoCallset
```

Comment:
aren't we getting rid of this branch now? Do we want to remove it from all of the workflows?

Reply:
Done